UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Models/Response/SalesCatalogueDataProductResponse.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace UKHO.SalesCatalogueFileShareServicesMock.API.Models.Response
+{
+    public class SalesCatalogueDataProductResponse
+    {
+        public short BaseCellEditionNumber { get; set; }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public DateTime BaseCellIssueDate { get; set; }
+        public string BaseCellLocation { get; set; }
+        public int? BaseCellUpdateNumber { get; set; }
+        public List<string> CancelledCellReplacements { get; set; }
+        public int? cancelledEditionNumber { get; set; }
```

Comment:
Can property names for cancelledEditionNumber, dataCoverageCoordinates, id, lastUpdateNumberPreviousEdition start with capital letter to maintain consistency with other property's casing.